Patched Sur/KextPatchView.swift

```diff
@@ -52,8 +52,27 @@ struct ButtonsView: View {
         } else if p == 3 {
             RunActionsDisplayView(action: {
                 do {
-                    try shellOut(to: "echo '\(password)' | sudo -S '/Volumes/Install macOS Big Sur Beta/patch-kexts.sh'")
-                    p = 4
+                    let model = (try? shellOut(to: "sysctl -n hw.model")) ?? "MacModelX,Y"
+                    if model == "MacBookPro8,1"{
+                        try shellOut(to: "echo '\(password)' | sudo -S '/Volumes/Install macOS Big Sur Beta/patch-kexts.sh --2011'")
+                        p = 4
+                    }
+                    else if model == "MacBookAir4,2"{
+                        try shellOut(to: "echo '\(password)' | sudo -S '/Volumes/Install macOS Big Sur Beta/patch-kexts.sh --2011'")
+                        p = 4
+                    }
+                    else if model == "iMac12,2"{
+                        try shellOut(to: "echo '\(password)' | sudo -S '/Volumes/Install macOS Big Sur Beta/patch-kexts.sh --2011'")
+                        p = 4
+                    }
+                    else if model == "iMac12,1"{
+                        try shellOut(to: "echo '\(password)' | sudo -S '/Volumes/Install macOS Big Sur Beta/patch-kexts.sh --2011'")
+                        p = 4
+                    }
+                    else {
+                        try shellOut(to: "echo '\(password)' | sudo -S '/Volumes/Install macOS Big Sur Beta/patch-kexts.sh'")
+                        p = 4
+                    }
```

Comment:
```suggestion
                    try shellOut(to: "echo '\(password)' | sudo -S '/Volumes/Install macOS Big Sur Beta/patch-kexts.sh'")
                    p = 4
```
Since patch-kexts auto-detects the Mac model, this isn't necessary.